### PR TITLE
Add Materialize Template

### PR DESCRIPTION
**Pull request by lb-**
Relates #18

### CODE REVIEW

**Comment by matteodem** [comment]
Got a screenshot or the atmosphere packages you used to have a look at it? :) Thanks a lot!

**Comment by lb-** [comment]
https://atmospherejs.com/materialize/materialize

Note that this is a stand along Material framework, different to a Material styled Bootstrap (or Semantic-ui).

File: lib/templates.html [hunk 1]
```diff
@@ -44,6 +44,42 @@ <h3 class="ui header">{{ take key='title' default='Comments'}}</h3>
...
+
+                  <div class="row">
+                      <div class="col s12">
+                        {{#each comment}}
+                            {{> commentsSingleComment }}
+                        {{/each}}
+                      </div>
+                  </div>
+
+                  {{#if hasMoreComments}}
```
Comment (matteodem) [inline lib/templates.html]: there is only one load more comments text needed.

**Comment by matteodem** [comment]
Doesn't look like replies are working right now. When you refactored the load more comments part I'll merge it and have a look at that.

**Comment by matteodem** [comment]
Replies look kinda weird too http://screencast.com/t/vE5AWl64r

**Comment by thomasandersen01** [comment]
Any news on this?

**Comment by matteodem** [comment]
I'll close this PR as there has not been any updates after I left my feedback on some of the code changes.

**Pull request closed without merging**